app/javascript/hooks/mutations/rooms/useUploadPresentation.jsx

```diff
@@ -0,0 +1,35 @@
+import { useMutation, useQueryClient } from 'react-query';
+import axios from 'axios';
+
+export default function useUploadPresentation(friendlyId) {
+  const queryClient = useQueryClient();
+
+  const uploadPresentation = (data) => {
+    const formData = new FormData();
+    formData.append('presentation', data);
+    axios.patch(`/api/v1/rooms/${friendlyId}.json`, formData);
+  };
+
+  const delay = (time) => new Promise((resolve) => {
+    setTimeout(resolve, time);
+  });
```

Comment:
We shouldn't be doing this anymore. This was required because of bad syntax. Check the other mutations